packages/api/src/graphql/client.ts

```diff
@@ -1,9 +1,31 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
-import config from '../helper/config';
+import pkg from "@apollo/client";
+const { ApolloClient, InMemoryCache, HttpLink, ApolloLink } = pkg;
+import { setContext } from "@apollo/client/link/context/index.js";
+import { removeTypenameFromVariables } from "@apollo/client/link/remove-typename/index.js";
+import { getToken } from "../authentication/jwt-token.js";
+import { config } from "../helper/config.js";
+
+const httpLink = new HttpLink({
+  uri: config.AAS_URI,
+});
+
+const authLink = setContext(async (_, { headers }) => {
```

Comment:
It looks like a problematic helper function, we are changing the context with this function. Instead, I'm prefer class based approach.

```
class RequestWrapper {
   public setHeader(k,v) {
       this.headers[k] = v,
   }
   
   public authorizedRequest() {
        return request({
             headers: {
                  ...this.headers,
                 // Your customized header here
             }
        });
   }
}
```